float64 SimTim      # 仿真时间
uint32 ID           # 目标ID
float64 x           # x方向相对坐标
float64 y           # y方向相对坐标
float64 v_x         # x方向相对速度
float64 v_y         # y方向相对速度
float64 a_x         # x方向相对加速度
float64 a_y         # y方向相对加速度
float64 xg	        # x方向绝对坐标
float64 yg          # y方向绝对坐标
float64 v_xg        # x方向绝对速度
float64 v_yg        # y方向绝对速度
float32 heading	    # 目标物绝对航向角
float64 length	    # 目标物长度
float64 width       # 目标物宽度
float64 height      # 目标物高度
uint8 type          # 目标物类型
float64 xrel	    # 目标物x方向绝对坐标减自车x方向绝对坐标
float64 yrel        # 目标物y方向绝对坐标减自车y方向绝对坐标
float64 v_xrel      # 目标物x方向绝对速度减自车x方向绝对速度
float64 v_yrel      # 目标物y方向绝对速度减自车y方向绝对速度
